var/spack/repos/builtin/packages/py-abipy/package.py

```diff
@@ -0,0 +1,79 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class PyAbipy(PythonPackage):
+    """Set of python modules and scripts to analyze the results of
+    ABINIT computations."""
+
+    homepage = "https://github.com/gmatteo/abipy"
+
+    version('master', git="https://github.com/gmatteo/abipy.git", branch='master')
```

Comment:
> I noticed that you wrote our abinit package. Thank you! A lot of our users at ANL use abinit,  and have requested abipy as well.

Thanks, I'm happy to hear that!
I'm actively involved in the development of Abinit (and abipy) so feel free to
contact me if there are portability issues with the versions of Abinit used at ANL.

> noticed that there were no versions available for download on PyPI. Let me know when a version is released and I can add it instead of cloning the latest master branch, which may or may not be broken (currently broken).

We haven't released an official version of abipy on PyPi because there's been a significant refactoring in Abinit 
related to the migration to hdf5 + netcdf4 and the introduction of new netcdf file formats.
We've recently released abinit 8.2.2 and this is the first version that will be officially supported by abipy. 
I plan to release a stable version of abipy before the end of the next week.
I'm still collecting contributions from other abipy developers and I have to send a pull request to pymatgen to have my last modifications merged (at present, abipy requires my fork of the pymatgen repo) 
After this merge, we can freeze the developments and we can deploy to production.